README.md

```diff
@@ -22,19 +22,22 @@ The `master` branch is now dedicated to development of the `3.3.x` line.
 With Gradle from repo.spring.io or Maven Central repositories (stable releases only):
 
 ```groovy
-    repositories {
-//      maven { url 'https://repo.spring.io/snapshot' }
-      maven { url 'https://repo.spring.io/milestone' }
-      mavenCentral()
-    }
-
-    dependencies {
-      //compile "io.projectreactor:reactor-core:3.4.0-SNAPSHOT"
-      //testCompile("io.projectreactor:reactor-test:3.4.0-SNAPSHOT")
-      //TODO change to the release artifact and comment snapshot repo above when GA
-      compile "io.projectreactor:reactor-core:3.4.0-M1"
-      testCompile("io.projectreactor:reactor-test:3.4.0-M1")
-    }
+repositories {
+    mavenCentral()
+    maven { url "https://repo.spring.io/milestone" }
+
+    // Snapshots
+    // maven { url "https://repo.spring.io/snapshot" }
+}
+
+dependencies {
+    compile "io.projectreactor:reactor-core:3.4.0-M1"
+    testCompile "io.projectreactor:reactor-test:3.4.0-M1"
+
+    // Snapshots
+    // compile "io.projectreactor:reactor-core:3.4.0-SNAPSHOT"
+    // testCompile "io.projectreactor:reactor-test:3.4.0-SNAPSHOT"
```

Comment:
I’ve removed the TODO but please correct me if I’m wrong. I understood the declaration that there are snapshots if someone wants to use them.